standalone/header-footer/src/StandaloneProvider/StandaloneProvider.js

```diff
@@ -31,7 +31,9 @@ export default class StandaloneProvider extends Component {
 
     return (
       <div className={cssSelectorPrefix.replace('.', '')}>
-        <StandaloneComponent {...componentProps} />
+        <div className={styles.root}>
```

Comment:
Whats the reason for needing this second `div` element, it's not immediately obvious?

Reply:
Ah, nice catch, I forgot to call this out.

I actually found an issue that we didn't include the provider styles in the standalone components, which meant that everything was in Times New Roman, etc. The provider styles need to be attached to a div nested within the existing div so that we can scope them to the `cssSelectorPrefix`.

As part of this PR, I extracted the provider styles into a mixin so that it can be used across both Provider components.

I'll update the description with this detail.